src/sap/fhir/model/r4/lib/FHIRRequestor.js

```diff
@@ -232,10 +231,10 @@ sap.ui.define([
 		var sRequestUrl;
 		var sContentType = "application/json";
 		var oBindingInfo = this.oModel.getBindingInfo(oFHIRUrl.getRelativeUrlWithoutQueryParameters());
-		// if the method is GET , secure search is enabled
+		// if the method is GET, secure search is enabled
```

Comment:
comments are no longer necessary, isn't?

Reply:
lets put the comments inside the method implementation if needed

Reply:
okay

Reply:
done